test/examples/dataviz/c3/index.html

```diff
@@ -174,18 +180,28 @@
     .title('Percentage of awesome')
     .render();
 
-  var c3donut = new Keen.Dataviz()
-    //.library('c3')
-    //.chartType('donut')
-    .el(document.getElementById('c3-donut'))
-    .parseRawData({ result: [
-        { label: 'A', result: 9 },
-        { label: 'B', result: 7 },
-        { label: 'C', result: 4 }
-      ]
-    })
-    //.title('Best Donut!')
-    .render();
+  // var c3donut = new Keen.Dataviz()
+  //   //.library('c3')
+  //   //.chartType('donut')
+  //   .el(document.getElementById('c3-donut'))
+  //   .parseRawData({ result: [
+  //       { label: 'A', result: 9 },
+  //       { label: 'B', result: 7 },
+  //       { label: 'C', result: 4 }
+  //     ]
+  //   })
+  //   //.title('Best Donut!')
+  //   .render();
+
+  client.draw(count, document.getElementById('c3-donut'), {
+    library: 'c3',
+    chartType: 'donut',
+    chartOptions: {
+      test: function(a,b,c){
```

Comment:
This was part of validating the fix, but sure - probably best to remove this so it doesn't cause any confusion :+1: